modules/login/php/requestaccount.class.inc

```diff
@@ -61,14 +60,6 @@ class RequestAccount extends \NDB_Form
         $this->addCheckbox("examiner", "Examiner role", array());
         $this->addCheckbox("radiologist", "Radiologist", array());
 
-        // Get reCATPCHA keys
-        $reCAPTCHAPrivate = $config->getSetting('reCAPTCHAPrivate');
```

Comment:
I deleted references to reCAPTCHA here as a temporary measure because many of the variables relating to its error reporting were marked as unused. @maltheism [discovered](https://github.com/aces/Loris/pull/4395#discussion_r265660761) that this feature hasn't been working in LORIS for some time now anyway so I don't think there's any harm done here. A separate issue/PR will be created by Alizee to address the future of reCAPTCHA in LORIS.